test/correctness/simd_op_check_xtensa.cpp

```diff
@@ -140,11 +140,11 @@ class SimdOpCheckXtensa : public SimdOpCheckTest {
         // These are not generated right now, because vectors are split now, so comment out for now.
         // Narrowing with shifting.
         // check("halide_xtensa_narrow_with_shift_i16", vector_width / 2, i16(i32_1 >> i32_2));
-        // check("halide_xtensa_narrow_with_shift_i16", vector_width / 2, i16(i32_1 / 4));
+        check("halide_xtensa_narrow_with_shift_i16", vector_width / 2, i16(i32_1 / 4));
         // check("halide_xtensa_narrow_with_shift_u16", vector_width / 2, u16(i32_1 >> i32_2));
-        // check("halide_xtensa_narrow_with_shift_u16", vector_width / 2, u16(i32_1 / 4));
+        check("halide_xtensa_narrow_with_shift_u16", vector_width / 2, u16(i32_1 / 4));
 
-        // check("IVP_AVGshouldhavefailedRNX16", vector_width / 2, i16((i32(i16_1) + i32(i16_2) + 1) / 2));
```

Comment:
Why should it have failed?

Reply:
I think I was trying to test that it's actually will fail, if that string is not found and forgot to revert it back.